### PR TITLE
Use target bed files as regions instead of bait intervals for SNV calling in WES samples

### PR DESCRIPTION
**Pull request by ramprasadn**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/raredisease/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/raredisease _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test_one_sample,docker --outdir <OUTDIR>`).
- [ ] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Comment by jemten** [comment]
I tried running a test case and the bed file presented to deepvariant can't be bgzipped. 
````
I1018 19:16:53.355009 139642998064960 genomics_reader.py:222] Reading ADM997A3_sorted_md.bam with NativeSamReader
I1018 19:16:53.406375 139642998064960 make_examples_core.py:301] Task 18/36: Preparing inputs
I1018 19:16:53.513127 139642998064960 genomics_reader.py:222] Reading ADM997A3_sorted_md.bam with NativeSamReader
I1018 19:16:53.529938 139642998064960 make_examples_core.py:301] Task 18/36: Common contigs are ['1', '2', '3', '4', '5', '6', '7', '8', '9', '10', '11', '12', '13', '14', '15', '16', '17', '18', '19', '20', '21', '22', 'X', 'Y', 'MT']
Traceback (most recent call last):
  File "/home/proj/stage/analysis/cases/cleanshrimp/work/Bazel.runfiles_h0i4nyq0/runfiles/com_google_deepvariant/deepvariant/make_examples.py", line 234, in <module>
    app.run(main)
  File "/home/proj/stage/analysis/cases/cleanshrimp/work/Bazel.runfiles_h0i4nyq0/runfiles/absl_py/absl/app.py", line 312, in run
    _run_main(main, args)
  File "/home/proj/stage/analysis/cases/cleanshrimp/work/Bazel.runfiles_h0i4nyq0/runfiles/absl_py/absl/app.py", line 258, in _run_main
    sys.exit(main(argv))
  File "/home/proj/stage/analysis/cases/cleanshrimp/work/Bazel.runfiles_h0i4nyq0/runfiles/com_google_deepvariant/deepvariant/make_examples.py", line 224, in main
    make_examples_core.make_examples_runner(options)
  File "/home/proj/stage/analysis/cases/cleanshrimp/work/Bazel.runfiles_h0i4nyq0/runfiles/com_google_deepvariant/deepvariant/make_examples_core.py", line 2739, in make_examples_runner
    regions, calling_regions = processing_regions_from_options(options)
  File "/home/proj/stage/analysis/cases/cleanshrimp/work/Bazel.runfiles_h0i4nyq0/runfiles/com_google_deepvariant/deepvariant/make_examples_core.py", line 2641, in processing_regions_from_options
    calling_regions = build_calling_regions(
  File "/home/proj/stage/analysis/cases/cleanshrimp/work/Bazel.runfiles_h0i4nyq0/runfiles/com_google_deepvariant/deepvariant/make_examples_core.py", line 607, in build_calling_regions
    ranges.RangeSet.from_regions(regions_to_include, contig_dict)
  File "/home/proj/stage/analysis/cases/cleanshrimp/work/Bazel.runfiles_h0i4nyq0/runfiles/com_google_deepvariant/third_party/nucleus/util/ranges.py", line 170, in from_regions
    return cls(ranges=from_regions(regions, contig_map=contig_map))
  File "/home/proj/stage/analysis/cases/cleanshrimp/work/Bazel.runfiles_h0i4nyq0/runfiles/com_google_deepvariant/third_party/nucleus/util/ranges.py", line 117, in __init__
    for i, range_ in enumerate(ranges):
  File "/home/proj/stage/analysis/cases/cleanshrimp/work/Bazel.runfiles_h0i4nyq0/runfiles/co
m_google_deepvariant/third_party/nucleus/util/ranges.py", line 509, in from_regions
    yield parse_literal(region, contig_map)
  File "/home/proj/stage/analysis/cases/cleanshrimp/work/Bazel.runfiles_h0i4nyq0/runfiles/com_google_deepvariant/third_party/nucleus/util/ranges.py", line 597, in parse_literal
    raise ValueError(
ValueError: Could not parse "home.bed.gz" as a region literal.  Region literals should have the form "chr:start-stop" or "chr:start" or just "chr".  A common error is to use the "chr" prefix on inputs that don't have it, or vice-versa.
```
We can use `ch_target_bed_unprocessed` or we have to decompress the bed file again prior to deepvariant.

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### `Changed`
 
 - Suffix used to identify unique fastq pairs from "\_T" to "\_LNUMBER" [#638](https://github.com/nf-core/raredisease/pull/638)
+- Change the regions used by deepavariant for WES samples from bait interval files to target bed files [#636](https://github.com/nf-core/raredisease/pull/636)
```
Comment (jemten) [inline CHANGELOG.md:15]: Shouldn't the changelog just list the changes from version 2.2.0 to the next? This is more a fix of PR #633

Comment (ramprasadn) [reply CHANGELOG.md:15]: Ah true.. I will remove it.

**Review by jemten — APPROVED** [review]
👍

**Pull request merged**